lombok.config

```diff
@@ -0,0 +1,3 @@
+config.stopbubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.anyConstructor.addConstructorProperties = true
```

Comment:
Why have we added lombok it has not been used. Also we decided not to use lombok on this project.